testing_and_setup/compass/ocean/hurricane/USDEQU120at30cr10rr2/synthetic_sandy/config_driver.xml

```diff
@@ -0,0 +1,8 @@
+<driver_script name="run_test.py">
+	<case name="init">
+		<step executable="./run.py" quiet="true" pre_message=" * Running init" post_message="     Complete"/>
+	</case>
+	<case name="forward">
```

Comment:
@sbrus89, should there be a spin-up case here?

Reply:
Definitely, I'll add that.